api/controllers/OpenEAppController.js

```diff
@@ -96,7 +100,31 @@ const OpenEAppController = {
     },
 
     _formatDate(date) {
-        return moment(date).format('DD MMMM YYYY');
+        return dayjs(date).format('DD MMMM YYYY');
+    },
+
+    _calculateDaysLeftToDownload(req) {
+        const urlObj = new URL(
+            `${req.protocol}://${req.headers.host}${req.originalUrl}`
```

Comment:
does this mean you'll accept HTTP requests too?

Reply:
yep, currently testing locally only has HTTP so I'll have to test on the AWS instance to see if it'll do HTTPS as well.